mappings/8622bfefea35d826ee5eded34ed3ac2310c3610df246095a81e499bc4c534e53.json

```diff
@@ -0,0 +1,64 @@
+{
+    "subject": "8622bfefea35d826ee5eded34ed3ac2310c3610df246095a81e499bc4c534e53",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://www.kraldano.nl",
```

Comment:
Hey @Krallez,

website can't be validated. If the page is not up, you can remove the field, it is optional. 
Please update the PR.

Thanks!